### PR TITLE
Some docs changes

### PR DESCRIPTION
**Pull request by SunDwarf**
Disclaimer: I do not know ruby

### CODE REVIEW

File: lib/discordrb/data.rb [hunk 1]
```diff
@@ -296,6 +296,9 @@ module PermissionCalculator
     # through for example being the server owner or having the Manage Roles permission
     # @param action [Symbol] The permission that should be checked. See also {Permissions::Flags} for a list.
     # @param channel [Channel, nil] If channel overrides should be checked too, this channel specifies where the overrides should be checked.
+    # @example Check if a member can send messages to a specific channel in a server.
```
Comment (meew0) [inline lib/discordrb/data.rb]: "a member" -> "the bot"

File: lib/discordrb/data.rb [hunk 2]
```diff
@@ -315,6 +318,8 @@ def permission?(action, channel = nil)
     # Manage Roles)
     # @param action [Symbol] The permission that should be checked. See also {Permissions::Flags} for a list.
     # @param channel [Channel, nil] If channel overrides should be checked too, this channel specifies where the overrides should be checked.
+    # @example Check if a member has the Manage Channels permission in the server.
```
Comment (meew0) [inline lib/discordrb/data.rb]: This should say "has the Manage Channels permission *defined* on the server" to differentiate between this and `permission?`

**Comment by meew0** [comment]
Otherwise, not bad.

**Comment by SunDwarf** [comment]
Fixed both

**Pull request merged**